charts/ingress-monitor-controller-endpoint/README.md

```diff
@@ -0,0 +1,24 @@
+# ingress-monitor-controller-endpoint
+
+![Version: 0.1.0](https://img.shields.io/badge/Version-0.1.0-informational?style=flat-square)
+
+A Helm chart to install Ingress Monitor Controller endpoints
+
+## Maintainers
+
+| Name | Email | Url |
+| ---- | ------ | --- |
+| Luis Gallardo | luis.gallardo@binbash.com.ar | https://binbash.com.ar |
```

Comment:
@lgallard Luis let's also add here leverage@binbash.com.ar to the maintainers list please, which is a distribution list that include all Leverage maintainers (you're also included here). Thanks!

```
## Maintainers

| Name | Email | Url |
| ---- | ------ | --- |
| Binbash Leverage Team | leverage@binbash.com.ar | https://leverage.binbash.com.ar |
| Luis Gallardo | luis.gallardo@binbash.com.ar | https://leverage.binbash.com.ar |
```

Reply:
Done!